spec/features/admin_claim_approve_spec.rb

```diff
@@ -23,8 +23,12 @@
 
         claim_to_approve.reload
 
-        expect(claim_to_approve.approved_at).to eq(Time.zone.now)
-        expect(claim_to_approve.approved_by).to eq("12345")
+        expect(claim_to_approve.checks.count).to eq(1)
+
+        check = claim_to_approve.checks.last
```

Comment:
Probably a good idea to pull this out into a method, so we're consistent, and know why we're checking the last. It should really be ordered by creation_date, in case the list is populated differently, too, shouldn't it?

Reply:
Really like this. Do we think it is the `last_check` as a check is only created when performing an action?

Reply:
`latest_check`, maybe? Since it will be ordered by date.